README.md

```diff
@@ -27,11 +33,18 @@ cd qtcodes
 pip install --upgrade .
 ```
 
-To check if the installation was successful, run:
+If you also want to download the dependencies needed to run optional tutorials, please use `pip install --upgrade .[dev]` or `pip install --upgrade '.[dev]'` (for `zsh` users).
 
-```python
->>> import qtcodes as qtc
+
+#### Installation for Devs
+
+If you intend to contribute to this project, please install `qtcodes` in editable mode as follows:
+```bash
+git clone https://github.com/yaleqc/qtcodes.git
+cd qtcodes
+pip install -e .[dev]
```

Comment:
Never seen the `[dev]` part of this. What does that do? Is that for optional dependencies?

Reply:
Yes, it's for the extra dependences stored in `EXTRA_REQUIREMENTS["dev"]`:

https://github.com/yaleqc/qtcodes/blob/feature/59%2Fsims-pip/setup.py#L18